package.json

```diff
@@ -91,20 +65,30 @@
     "uuid": "^7.0.3"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.10",
+    "@babel/plugin-transform-regenerator": "^7.12.1",
+    "@babel/preset-env": "^7.12.11",
     "@typechain/ethers-v4": "^1.0.1",
+    "@types/chai": "^4.2.14",
+    "@types/chai-as-promised": "^7.1.3",
     "@types/jest": "^26.0.13",
     "@types/lodash.difference": "^4.5.6",
     "@types/uuid": "^8.3.0",
     "@typescript-eslint/eslint-plugin": "^4.0.1",
     "@typescript-eslint/parser": "^4.0.1",
+    "babel-jest": "^26.6.3",
     "bili": "^5.0.5",
+    "chai": "^4.2.0",
```

Comment:
While using `jest` you don't need chai anymore :) `jest` has built in assertion (expect), mocks etc :)

Reply:
Ok. thanks. will remove it